README.md

```diff
@@ -23,3 +23,32 @@ The code in this repository is released under the Apache License, see the [LICEN
 ## Contribution
 
 Please use [this guide](CONTRIBUTING.md) to make a contribution to the project and information to report any issues.
+
+## Development
+
+The roles developed for this playbook are tested with [Molecule](https://molecule.readthedocs.io/en/latest/).
+
+### Roles tests
+
+You can run test for each role by entering the role folder and running `molecule test`:
+
+```bash
+cd roles/activemq
+molecule test
+```
+
+### Integration tests
+
+On the root folder there is a molecule scenario to run the entire playbook on EC2 instances with different operating systems.
+
+Some environment variables are required to execute integration tests locally, please take a look at the [.envrc](.envrc) file.
+
+To have environment variables automatically loaded when entering the root folder, please install [direnv](https://direnv.net/).
```

Comment:
it's not a dependency, it's just an aid for who want to run ec2 tests locally on their own environment, maybe I can clarify it

however, I didn't elaborate more here because I think there are very few people interested in running integration tests but at least a pointer in the readme is good to have

Reply:
ok. Fine with me then as long as it's not required for builds